### ISSUE TITLE
Relations between two labels on one word causes completions to not load

### ISSUE THREAD

**Issue by ghanshyamdev** [heartexlabs/label-studio#315]
**Describe the bug**
When I create relations between two different labels on the same text cause completions to not load and in the console below error occurs:
```
Uncaught Error: [mobx-state-tree] Can only store references to tree nodes or identifiers, got: 'undefined'
    at be (main.4c9ff345.js:sourcemap:1)
    at new e (main.4c9ff345.js:sourcemap:1)
    at t.instantiate (main.4c9ff345.js:sourcemap:1)
    at main.4c9ff345.js:sourcemap:1
    at main.4c9ff345.js:sourcemap:1
    at Array.forEach (<anonymous>)
    at t.forAllProps (main.4c9ff345.js:sourcemap:1)
    at t.initializeChildNodes (main.4c9ff345.js:sourcemap:1)
    at new t (main.4c9ff345.js:sourcemap:1)
    at ae (main.4c9ff345.js:sourcemap:1)
```

**To Reproduce**
1. Create a "**Named Entity Recognition**" project 
2. Import sample task
3. Assign two different labels to the same text
4. Now select the first row from `Regions` and click on `Create Relation` and select another label from the assigned label
5. Press submit button
6. Reload the page and see the error appears in the console.

**Video**
The video below is very useful for reproducing the error
![Undefined_error](https://user-images.githubusercontent.com/45035063/84227917-463ee500-ab03-11ea-8114-fa3bb8c74acf.gif)

**Environment:**
 - OS: macOS, ubuntu
 - Browser Chrome, Firefox
 - Label studio frontend version: 0.7.0

**Any recommendations from you to fix this problem.**

**Comment by deppp** [comment]
Hey @ghanshyamdev thanks for reporting that, we're on it

**Comment by hlomzik** [comment]
Fixed by #312 and [released](https://github.com/heartexlabs/label-studio/releases/tag/v0.7.2).
Thank you!

**Comment by ghanshyamdev** [comment]
@hlomzik @deppp  still I am able to reproduce this issue in the master branch
